tests/canary/canary.buildspec.yaml

```diff
@@ -15,8 +14,15 @@ phases:
     commands:
       # Run tests
       - docker run --name kf-distro-canary $(env | cut -f1 -d= | sed 's/^/-e /') --mount type=bind,source="$(pwd)/",target="/kubeflow-manifests/" ${ECR_CACHE_URI}:latest
-      
-      # Push test image to cache ECR repo
+  post_build:
+    commands:
+      - docker cp kf-distro-canary:/kubeflow-manifests/tests/canary/integration_tests.xml /tmp/results.xml
```

Comment:
if test run fails, do the rest of the commands still run?

Reply:
the docker push command will still run because it has the `|| true`

Reply:
added.